File: Playgrounds.BackEnd/PlaygroundsGallery.Domain/Models/Photo.cs
namespace PlaygroundsGallery.Domain.Models
{
    public class Photo: Entity
    {
		public string Url { get; set; }
		public string PublicId { get; set; }
		public string Description { get; set; }
		public int MemberId { get; set; }
		public virtual Member Member { get; set; }
		public int? PlaygroundId { get; set; }
		public virtual Playground Playground { get; set; }
		public bool Deleted { get; set; } = false;
	}
}

File: Playgrounds.FrontEnd/src/app/_models/Photo.ts
export interface Photo {
    id: number;
    url: string;
    publicId: string;
    memberId?: number;
    memberLoginName?: string;
    playgroundId?: number;
    playgroundAddress?: string;
    playgroundLocationStr?: string;
    playgroundLocationId?: number;
    description?: string;
    created?: Date;
}

export interface PhotoToUpload {
    playgroundId?: number;
    description?: string;
}
